luaui/Widgets/gui_healthbars_gl4.lua

```diff
@@ -350,6 +350,13 @@ local minReloadTime = 4 -- weapons reloading slower than this willget bars
 local featureHealthVBO
 local featureResurrectVBO
 local featureReclaimVBO
+
+local minUiScale = 0.8    -- FIXME: Get these values from "Options" widget, respectively min and max value from 'uiscale'
```

Comment:
Unfortunately Options widget does two things that makes this difficult.

One is actually just getting the options values at the DrawScreen call and the other is initializing the children widgets at its own initialization (healthbars gl4 being one of them)

Reply:
I'm not really understanding the involvement of the min/max ui_scale the options slider is able to set.

Furthermore I'm not fully convinced ui_scale is the right var to apply to healthbar scaling (alone), but that I/we have yet to experience I guess.

Old healthbar just used its own configdata var and options had a separated slider to give the option to the user directly.

Reply:
Oh did it?

I'll have to dig for that, do we want to have a separate slider for healthbar scale?

Reply:
There it is?

https://github.com/beyond-all-reason/Beyond-All-Reason/commit/84dc7ae3c373f30fe86007abba3c99e75268a384#diff-a00e968bb5f7ffe2e9defd5da5fd978e14e45bdc0ebf3a7d7976995912094299L2848

Do we want to bring options back and separate for healthbars?

Reply:
In this case, should we perhaps port an bring all options?

Scale, drawDistance, hideHealthbars and variableSizes

Reply:
Seems like the only option that was not `dev` was scale, so perhaps only worth to port that one

Reply:
the others were hidden as dev options because we needed a sane amount of settings options and to not overload the player with hardly meaningful ones and trust the defaults.

variablesizes was a default enabled one for example we'd still like back

Reply:
So... should we bring back separate scaling or not?

Reply:
Done